chisel-wrapper

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# USAGE:
+#
+#   ./<script> [ chisel-cut-OPTIONS ] <slices..>
+#
+# Installs the slices, using
+#
+#   chisel cut "$@"
+#
+# Generates a status file similar to /var/lib/dpkg/status
+# at the CHISEL_DPKG_STATUS_FILE path.
+#
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -eu
+
+CHISEL_BIN="${CHISEL_BIN:-"chisel"}"
+CHISEL_CACHE_DIR="$(mktemp -d)"
+
+cleanup() {
+	rm -rf "$CHISEL_CACHE_DIR"
+}
+trap cleanup EXIT
+
+print_error() {
+	echo "Error:" "$@" >> /dev/stderr
```

Comment:
Why not just echo "Error:" "$@" >&2?

Reply:
Changed in https://github.com/canonical/rocks-toolbox/pull/1/commits/d0377af9488a00ac7455f4e3a256651ca7fffd7b. Thanks.